front/app/containers/ProjectsShowPage/shared/header/ProjectActionButtons.tsx

```diff
@@ -186,24 +190,15 @@ const ProjectActionButtons = memo<Props>(({ projectId, className }) => {
           />
         </SeeIdeasButton>
       )}
-      {showIdeasButton && (
+      {showIdeasButton && !hasProjectEnded && (
         <IdeaButton
           id="project-ideabutton"
           projectId={project.id}
           participationContextType="project"
+          phaseId={isPhaseIdeation ? currentPhase.id : ''}
```

Comment:
I think we might need a condition on the `participationContextType` above as well 🤔